test/unity/toolkits/drawing_classifier/test_dc_prediction.cxx

```diff
@@ -261,31 +262,34 @@ BOOST_AUTO_TEST_CASE(test_drawing_classifier_perform_inference) {
   log_for_debug("test_drawing_classifier_perform_inference");
 
   for (auto& entry : TEST_CASES) {
-    size_t batch_size = std::get<0>(entry);
-    size_t num_of_rows = std::get<1>(entry);
-    size_t num_of_classes = std::get<2>(entry);
+    for (bool is_bitmap_based : {true}) {
```

Comment:
you can edit the `TEST_CASES` to have a tuple of 4 elements.

Reply:
this line is a bad practice since it removes the non-bitmap-based test.

Reply:
to make myself clear, tuple of <flag_of_bitmap, batch_size, num_of_rows, num_of_classes>. which means you need to double the size of `TEST_CASES`.

Reply:
or a more simple way
```
prediction_test_driver(true, batch_size, num_of_rows, num_of_classes, runner);
prediction_test_driver(false, batch_size, num_of_rows, num_of_classes, runner);
```
Now I think I understand what you are going to do
```
for (bool is_bitmap_based : {true, false}) {
```
That's also okay, kind of a overkill.

Reply:
Addressed.

Reply:
please remove the outter loop ` for (bool is_bitmap_based : {true}) {` completely. 

You already have `is_bitmap_based` taken care of
```
for (bool is_bitmap_based : {true, false}) {
       prediction_test_driver(batch_size, num_of_rows, num_of_classes, runner,
                              is_bitmap_based);
     }
```